index.js

```diff
@@ -51,7 +51,9 @@ module.exports = opts => {
 	opts = opts || {};
 
 	if (process.platform === 'win32') {
-		return [];
+		let cp = require("child_process");
+		let dk = cp.spawnSync("doskey", ["/history"]);
```

Comment:
Use a more descriptive name than `dk`.

Reply:
Use the `encoding` option instead of `toString()`.

Reply:
Noted and changed.

`toString()` is required because stdout is a buffer.